MnemonicKit.xcodeproj/xcshareddata/xcschemes/MnemonicKit.xcscheme

```diff
@@ -28,26 +28,7 @@
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES">
       <Testables>
```

Comment:
This PR fails on CI: https://travis-ci.org/keefertaylor/MnemonicKit/builds/471070508

The command being used is: 
` xcodebuild -enableCodeCoverage=YES -scheme MnemonicKitTests -workspace MnemonicKit.xcworkspace -destination platform\=iOS\ Simulator,OS\=12.0,name\=iPhone\ X test`

and fails with: 
` : error: Unable to find a destination matching the provided destination specifier:
		{ platform:iOS Simulator, OS:12.0, name:iPhone X }`

I think the missing destination might be a red herring since the MnemonicKitTests scheme is deleted here.

Reply:
Should work now.